packages/app/src/components/Root/Root.tsx

```diff
@@ -77,6 +79,7 @@ export const Root = ({ children }: PropsWithChildren<{}>) => (
       </SidebarGroup>
       <SidebarSpace />
       <SidebarDivider />
+      <Administration />
```

Comment:
https://github.com/janus-idp/backstage-plugins/pull/895/files#r1386635365 - Same applies for this